<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calib</class>
 <widget class="QWidget" name="Calib">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="x_dir">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="y_dir">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="z_dir">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X-direction (mm)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>y-direction (mm)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>z-direction (mm)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushbutton_move">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_record">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>70</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start recording</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stop_record">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop recording</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_record_status">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>40</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Recording status</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pauseRecording">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>150</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause recording</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_setReference">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set reference position</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start_3D_sweep">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start 3D sweep</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_init_pos">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Move robot to initial position</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_GausstoOrigin">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Move Gauss-Probe to Origin</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
